include/nccl_ofi_mr.h

```diff
@@ -98,9 +104,23 @@ static inline uintptr_t nccl_ofi_mr_ckey_len(nccl_ofi_mr_ckey_ref ckey)
 	}
 }
 
+static inline void nccl_ofi_mr_ckey_round(int fd, uint64_t offset, size_t *len, void **base_addr, const char *type)
```

Comment:
i'm not sure why we're logging the FD or offset here.  We were fine without that before.

Reply:
Removed fd and offest.